sli/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ target_link_libraries( sli_readline sli_lib nestutil ${READLINE_LIBRARIES} )
 
 # add the executable
 add_executable( sli puresli.cc )
-target_link_libraries( sli sli_lib sli_readline )
+target_link_libraries( sli sli_lib sli_readline ${GSL_LIBRARIES} )
```

Comment:
why is this addition necessary?

Reply:
This is related to:

> I tested with cmake 2.8.12, 3.5.1 (ubuntu 16.04) and 3.9.4 (OS X). On the ubuntu system I encountered unrelated compilation issues with gsl and fixed them with 2411f5e. I also tested building and loading `MyModule` installed into a custom location.

Apparently, cmake 2.8.12 cannot do the transitive library discovery:
```
nestutil requires gsl
sli requires nestutil, hence slightly requires gsl
```
and does not put gsl into linkage with sli...